ethdb/kv/kv_snapshot.go

```diff
@@ -347,20 +397,24 @@ func (s *snTX) CollectMetrics() {
 }
 
 func (s *snTX) getSnapshotTX(bucket string) (ethdb.Tx, error) {
-	tx, ok := s.snTX[bucket]
-	if ok {
-		return tx, nil
-	}
-	sn, ok := s.snapshots[bucket]
-	if !ok {
+	var tx ethdb.Tx
+	switch bucket {
+	case dbutils.HeadersBucket:
+		tx = s.headersTX
+	case dbutils.BlockBodyPrefix:
+		fallthrough
```

Comment:
The same thing could be achieved by specifying a comma separated list in the `case` I believe